boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -1,13 +1,12 @@
-#include "fff.h"
-#include "gtest/gtest.h"
+#include <fff.h>
```

Comment:
Decided to include fff.h and gtest.h using `<>` becuase they're code we don't modify.